dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -69,6 +69,7 @@ type RemainingProps = {
 type Props = DefaultProps & (RightProps | InlineProps | RemainingProps);
 
 const labelHeight = constants.AD_LABEL_HEIGHT;
+const doubleMpuHeight = 600;
```

Comment:
Very minor and pedantic but whilst this ad slot size (300x600) is colloquially known as a 'double mpu' it is not exactly double of an mpu height (300x250)

https://thebannermen.com/banners/website/dmpu-ads/

https://s3.eu-west-2.amazonaws.com/s3-guardian-igad/files/pdf/Digital-Ad-Spec-2020_v2.pdf

It is also known as a 'half page' ad slot and this is the naming we give it in  `ad-sizes.ts` in commercial.

https://github.com/guardian/commercial/blob/01511c793cf4afe8b4fb39ecbc43e725cff8265b/src/core/ad-sizes.ts#L129

In which case you could do:

```suggestion
const halfPageHeight = adSizes.halfPage.height;
```

Not a blocker!